uimaj-core/src/main/java/org/apache/uima/cas/impl/package.html

```diff
@@ -30,7 +30,7 @@
   <body>
 <p>Implementation and Low-Level API for the CAS Interfaces.</p>
 <p>Internal APIs. Use these APIs at your own risk. APIs in this package are subject to change without notice, even in minor releases. Use of this package is not supported. If you think you have found a bug in this package, please try to reproduce it with the officially supported APIs before reporting it.</p>
-<hr />
+<hr>
```

Comment:
please run the javadoc with java 9 / 10 to see if any of these closing tags get complaints...

Reply:
JavaDoc 9/10 (and actually I think even 8) do not like self-closing HTML elements.

Reply:
+1